assets/xml/objects/object_fr.xml

```diff
@@ -1,67 +1,68 @@
+        <DList Name="gFrogLeftHeelDL" Offset="0xABA8" />
+        <DList Name="gFrogLeftToeDL" Offset="0xAE38" />
+        <DList Name="gFrogLeftShinDL" Offset="0xAED8" />
+        <DList Name="gFrogLeftThighDL" Offset="0xB0F8" />
+        <DList Name="gFrogDustMaterialDL" Offset="0xB328" />
+        <DList Name="gFrogDustModelDL" Offset="0xB338" />
+        <Limb Name="gFrogLowerBodyLimb" Type="Standard" EnumName="FROG_LIMB_LOWER_BODY" Offset="0xB3C8" />
+        <Limb Name="gFrogUpperBodyLimb" Type="Standard" EnumName="FROG_LIMB_UPPER_BODY" Offset="0xB3D4" />
+        <Limb Name="gFrogHeadRootLimb" Type="Standard" EnumName="FROG_LIMB_HEAD_ROOT" Offset="0xB3E0" />
+        <Limb Name="gFrogHeadLimb" Type="Standard" EnumName="FROG_LIMB_HEAD" Offset="0xB3EC" />
+        <Limb Name="gFrogRightIrisLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_IRIS" Offset="0xB3F8" />
+        <Limb Name="gFrogJawLimb" Type="Standard" EnumName="FROG_LIMB_JAW" Offset="0xB404" />
+        <Limb Name="gFrogRightEyeLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_EYE" Offset="0xB410" />
+        <Limb Name="gFrogLeftEyeLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_EYE" Offset="0xB41C" />
+        <Limb Name="gFrogLeftIrisLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_IRIS" Offset="0xB428" />
+        <Limb Name="gFrogLeftUpperArmLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_UPPER_ARM" Offset="0xB434" />
+        <Limb Name="gFrogLeftForearmLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_FOREARM" Offset="0xB440" />
+        <Limb Name="gFrogLeftHandLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_HAND" Offset="0xB44C" />
+        <Limb Name="gFrogRightUpperArmLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_UPPER_ARM" Offset="0xB458" />
+        <Limb Name="gFrogRightForearmLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_FOREARM" Offset="0xB464" />
+        <Limb Name="gFrogRightHandLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_HAND" Offset="0xB470" />
+        <Limb Name="gFrogRightThighLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_THIGH" Offset="0xB47C" />
+        <Limb Name="gFrogRightShinLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_SHIN" Offset="0xB488" />
+        <Limb Name="gFrogRightHeelLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_HEEL" Offset="0xB494" />
+        <Limb Name="gFrogRightToesLimb" Type="Standard" EnumName="FROG_LIMB_RIGHT_TOES" Offset="0xB4A0" />
+        <Limb Name="gFrogLeftThighLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_THIGH" Offset="0xB4AC" />
+        <Limb Name="gFrogLeftShinLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_SHIN" Offset="0xB4B8" />
+        <Limb Name="gFrogLeftHeelLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_HEEL" Offset="0xB4C4" />
+        <Limb Name="gFrogLeftToesLimb" Type="Standard" EnumName="FROG_LIMB_LEFT_TOES" Offset="0xB4D0" />
+        <Skeleton Name="gFrogSkel" Type="Flex" LimbType="Standard" LimbNone="FROG_LIMB_NONE" LimbMax="FROG_LIMB_MAX" EnumName="FrogLimbs" Offset="0xB538" />
```

Comment:
```suggestion
        <Skeleton Name="gFrogSkel" Type="Flex" LimbType="Standard" LimbNone="FROG_LIMB_NONE" LimbMax="FROG_LIMB_MAX" EnumName="FrogLimb" Offset="0xB538" />
```
We like to leave the EnumName as singular, since it usually refers to a single limb element rather than the collection of them all.

Reply:
oh lol ic i left it like that cus its plural in the object decomp example that makes hella sense tho ill fix that

Reply:
Ah, let me fix that. I think that's from before we decided to switch to singular. Thanks for bringing it up!